requirements-dev.txt

```diff
@@ -40,7 +40,6 @@ async-timeout==4.0.3
     # via
     #   -c requirements.txt
     #   aiokatcp
```

Comment:
Looks like it's still in requirements.txt based on this?
(the aiokatcp dependency can probably wait for another day)

Reply:
Yes, it's only removed as a direct requirement. I expect it to still be pulled in indirectly for the foreseeable future.